dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -79,6 +79,8 @@ def __init__(self, options: NumericalOptions = None) -> None:
         self._mode_is_enabled: bool = True
         self.num_zeros: int = 0
         self.num_negatives: int = 0
+        self._num_quantiles: int = 1000  # TODO: add to options
```

Comment:
good TODO here -- can you add an issue and link as well?

Reply:
https://github.com/capitalone/DataProfiler/issues/853